File: benchmarks/suite/array_fill.go
package main
import ("fmt";"time")
func main(){
  t0:=time.Now()
  var n int64=1000000; var rounds int64=200
  a:=make([]int64,n); var chk int64=0
  for r:=int64(0);r<rounds;r++{
    for i:=int64(0);i<n;i++{ a[i]=i*2654435761+r }
    chk^=a[(r*7)%n]
  }
  el:=time.Since(t0).Seconds()
  fmt.Println(chk); fmt.Printf("elapsed: %.6fs\n",el)
}

File: benchmarks/suite/array_fill.py
import time
t0 = time.perf_counter()
n = 1000000
rounds = 200
a = [0] * n
chk = 0
for r in range(rounds):
    for i in range(n):
        a[i] = i * 2654435761 + r
    chk ^= a[(r * 7) % n]
t1 = time.perf_counter()
print(chk)
print("elapsed: %.6fs" % (t1 - t0))
